Application crash due to missing view

### Steps to reproduce

1. Launch the application.
2. See application crash.

### What did you expect?

Normal application launch.

### What happened?

Application Crash due to missing view - waiting view.


CI build for this [commit](https://github.com/vector-im/element-android/commit/377c4e00a342b88a788847ae38dd99f3174aade1) .


**Stactkrace**
```
FATAL EXCEPTION: main
Process: im.vector.app.debug, PID: 30117
java.lang.RuntimeException: Unable to start activity ComponentInfo{im.vector.app.debug/im.vector.app.features.home.HomeActivity}: java.lang.NullPointerException: Missing required view with ID: im.vector.app.debug:id/waiting_view
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2951)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3086)
	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:78)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1816)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loop(Looper.java:193)
	at android.app.ActivityThread.main(ActivityThread.java:6718)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:491)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
Caused by: java.lang.NullPointerException: Missing required view with ID: im.vector.app.debug:id/waiting_view
	at im.vector.app.databinding.MergeOverlayWaitingViewBinding.bind(MergeOverlayWaitingViewBinding.java:106)
	at im.vector.app.databinding.ActivityHomeBinding.bind(ActivityHomeBinding.java:103)
	at im.vector.app.databinding.ActivityHomeBinding.inflate(ActivityHomeBinding.java:69)
	at im.vector.app.databinding.ActivityHomeBinding.inflate(ActivityHomeBinding.java:59)
	at im.vector.app.features.home.HomeActivity.getBinding(HomeActivity.kt:152)
	at im.vector.app.features.home.HomeActivity.getBinding(HomeActivity.kt:87)
	at im.vector.app.core.platform.VectorBaseActivity.onCreate(VectorBaseActivity.kt:234)
	at im.vector.app.features.home.HomeActivity.onCreate(HomeActivity.kt:171)
	at android.app.Activity.performCreate(Activity.java:7144)
	at android.app.Activity.performCreate(Activity.java:7135)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1271)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2931)
```


Please take a look @bmarty.

### Your phone model

_No response_

### Operating system version

_No response_

### Application version and app store

Element (F-Droid)  debug build - https://buildkite.com/matrix-dot-org/element-android/builds/3251#e9e0255a-337b-455f-a5a1-9a172bbaac91

### Homeserver

matrix.org

### Have you submitted a rageshake?

No

--- a/vector/src/main/res/layout/merge_overlay_waiting_view.xml
+++ b/vector/src/main/res/layout/merge_overlay_waiting_view.xml
@@ -1,72 +1,66 @@
 <?xml version="1.0" encoding="utf-8"?>
-<merge xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/waiting_view"
     android:layout_width="match_parent"
-    android:layout_height="match_parent">
+    android:layout_height="match_parent"
+    android:background="?vctr_waiting_background_color"
+    android:clickable="true"
+    android:focusable="true"
+    android:visibility="gone"
+    tools:visibility="visible">
 
-    <androidx.constraintlayout.widget.ConstraintLayout
-        android:id="@+id/waiting_view"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:background="?vctr_waiting_background_color"
-        android:clickable="true"
-        android:focusable="true"
-        android:visibility="gone"
-        tools:visibility="visible">
+    <LinearLayout
+        android:id="@+id/waiting_view_content"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_margin="@dimen/layout_horizontal_margin"
+        android:background="?attr/colorBackgroundFloating"
+        android:orientation="vertical"
+        android:padding="@dimen/layout_horizontal_margin"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintWidth_percent="@dimen/dialog_width_ratio">
 
         <LinearLayout
-            android:id="@+id/waiting_view_content"
-            android:layout_width="0dp"
+            android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:layout_margin="@dimen/layout_horizontal_margin"
-            android:background="?attr/colorBackgroundFloating"
-            android:orientation="vertical"
-            android:padding="@dimen/layout_horizontal_margin"
-            app:layout_constraintBottom_toBottomOf="parent"
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toTopOf="parent"
-            app:layout_constraintWidth_percent="@dimen/dialog_width_ratio">
-
-            <LinearLayout
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:orientation="horizontal">
-
-                <ProgressBar
-                    android:id="@+id/waitingCircularProgress"
-                    android:layout_width="40dp"
-                    android:layout_height="40dp"
-                    android:layout_gravity="center_vertical"
-                    android:layout_marginEnd="6dp" />
-
-                <TextView
-                    android:id="@+id/waitingStatusText"
-                    style="@style/Widget.Vector.TextView.Body"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:layout_gravity="center_vertical"
-                    android:textColor="?vctr_content_secondary"
-                    android:visibility="gone"
-                    tools:text="Waiting status..."
-                    tools:visibility="visible" />
-
-            </LinearLayout>
+            android:orientation="horizontal">
 
             <ProgressBar
-                android:id="@+id/waitingHorizontalProgress"
-                style="@style/Widget.Vector.ProgressBar.Horizontal"
-                android:layout_width="match_parent"
-                android:layout_height="20dp"
-                android:layout_marginTop="10dp"
+                android:id="@+id/waitingCircularProgress"
+                android:layout_width="40dp"
+                android:layout_height="40dp"
+                android:layout_gravity="center_vertical"
+                android:layout_marginEnd="6dp" />
+
+            <TextView
+                android:id="@+id/waitingStatusText"
+                style="@style/Widget.Vector.TextView.Body"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_gravity="center_vertical"
+                android:textColor="?vctr_content_secondary"
                 android:visibility="gone"
-                tools:max="100"
-                tools:progress="30"
+                tools:text="Waiting status..."
                 tools:visibility="visible" />
 
         </LinearLayout>
 
-    </androidx.constraintlayout.widget.ConstraintLayout>
+        <ProgressBar
+            android:id="@+id/waitingHorizontalProgress"
+            style="@style/Widget.Vector.ProgressBar.Horizontal"
+            android:layout_width="match_parent"
+            android:layout_height="20dp"
+            android:layout_marginTop="10dp"
+            android:visibility="gone"
+            tools:max="100"
+            tools:progress="30"
+            tools:visibility="visible" />
 
-</merge>
+    </LinearLayout>
+
+</androidx.constraintlayout.widget.ConstraintLayout>

